docs/docs/modules/indexes/vector_stores/integrations/vectara.mdx

```diff
@@ -0,0 +1,44 @@
+---
+hide_table_of_contents: true
+---
+
+What is Vectara?
+
+**Vectara Overview:**
+- Vectara is developer-first API platform for building GenAI applications
+- To use Vectara - first [sign up](https://console.vectara.com/signup) and create an account. Then create a corpus and an API key for indexing and searching.
+- You can use Vectara's [indexing API](https://docs.vectara.com/docs/indexing-apis/indexing) to add documents into Vectara's index
+- You can use Vectara's [Search API](https://docs.vectara.com/docs/search-apis/search) to query Vectara's index (which also supports Hybrid search implicitly).
+- You can use Vectara's integration with LangChain as a Vector store or using the Retriever abstraction.
+
+## Installation and Setup
+To use Vectara with LangChainJS no special installation steps are required. 
+
+These can be provided as environment variables or in a .env file:
+- export `VECTARA_CUSTOMER_ID`="your_customer_id"
+- export `VECTARA_CORPUS_ID`="your_corpus_id"
+- export `VECTARA_API_KEY`="your-vectara-api-key"
+
+import CodeBlock from "@theme/CodeBlock";
+
+# Vectara Integration
+
+Vectara provides both a SOTA optimized vector store and its own embeddings, so you don't have to calculate embeddings directly within the chain.
+
+## Setup
```

Comment:
This entire Setup block should be removed. As we do not have a npm package right now. 
We just do plain old HTTP calls to index and/or query the corpus.